.claude/commands/design.md

```
@@ -0,0 +1,538 @@
+# 특정 레이어만 설계
+/design application "주문 취소"
+/design domain "주문 취소"
+/design persistence "주문 취소"
+/design rest-api "주문 취소"
+```
+
+---
+
+## 실행 프로세스
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)